libr/util/str.c

```diff
@@ -3564,6 +3565,20 @@ R_API const char *r_str_str_xy(const char *s, const char *word, const char *prev
 	return d;
 }
 
+R_API size_t r_charset_encode_str(ut8 *asciistr, ut8 *in, size_t len_input, charset *custom_charset, size_t len_charset) {
+	size_t i, j;
+	for (i = 0; i < len_input; i ++) {
+		for (j = 0; j < len_charset; j ++) {
+			//if we want an hashtable, let's do: asciistr[i] = custom_charset[j].ch; We are currently debating.
+			if (in[i] == custom_charset[j].hx) {
```

Comment:
if this never happens in the loop, is this function to be considered to have failed?

Reply:
@crowell No. In case the plugin does not handle the opcode, then the opcode will be printed... Hmm... It is not the current behavior. I should change that.